annotation-solr/src/main/java/eu/europeana/annotation/solr/service/impl/SolrAnnotationServiceImpl.java

```diff
@@ -508,10 +508,10 @@ public List<String> checkDuplicateAnnotations(Annotation anno, boolean noSelfDup
 			query=solrUniquenessQueryTranscriptions(anno, noSelfDupplicate);
 		    break;
 		case CAPTIONING :
-			query=solrUniquenessQueryCaptions(anno, noSelfDupplicate);
+			query=solrUniquenessQueryCaptionsAndSubtitles(anno, noSelfDupplicate);
```

Comment:
for the captioning we still need an additional check, to ensure that there is only one caption for the resources. (It is not allowed to have two captions in 2 different languages). But this check cannot be implemented here. The additional check needs to be added in the caller method...